addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4228,12 +4243,12 @@ registry.BackgroundPosition = SnippetOptionWidget.extend({
      * @see this.selectClass for params
      */
     backgroundType: async function (previewMode, widgetValue, params) {
-        const backgroundType = async (context)=> {
-            await this.editorHelpers.setClass(context, this.$target[0], 'o_bg_img_opt_repeat', widgetValue === 'repeat-pattern');
-            await this.editorHelpers.setStyle(context, this.$target[0], 'background-position', '');
-            await this.editorHelpers.setStyle(context, this.$target[0], 'background-size', '');
-        }
-        await this.wysiwyg.execCommand(backgroundType);
+        console.log('backgroundType');
```

Comment:
Don't forget to fix this and all the others like it... :p

Reply:
Bien sure :)